### DISCUSSION TITLE
Validation pour rejoindre l'île noire

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2509] [Nouvelles idées]
Bonjour,

Lorsque nous sommes sur le bateau, il suffit de faire un rapport pour rejoindre l'île. 
Il arrive souvent que des joueurs, qui attendent leurs coéquipiers pour faire l'île à plusieurs, fassent un rapport par mégarde,, empêchant les autres de les rejoindre.

Je propose d'ajouter une confirmation pour rejoindre l'île, similaire à la commande pour boire une potion, avec la possibilité de forcer l'action si nécessaire. 
Ou un bouton d'annulation permettant de rester sur le bateau.

**Comment by Procion06** [comment]
Bonne idée !
Une confirmation du style : "Vous êtes arrivés sur l'île, souhaitez-vous débarquer ?" "Oui/Pas tout de suite"
ça évite de se retrouver tout seul comme un con à faire l'île en laissant ses potes sur le carreau XD

**Comment by Easygamer01s** [comment]
Je trouve que c'est une excellente idée, car après plusieurs années de jeux, on se fait vieux et c'est compliqué de ne pas faire un rapport par automatisme..

Une alternative est qu'il serait possible de rejoindre le bateau si un équipier est dans les 5 minutes où on quitte l'île..

**Comment by BananePlantain** [comment]
et puis l'évenement d'arrivée sur l'ile avec comme issues : "foncer sur l'ile" et "demander + d'infos" peut nous faire penser qu'on aura vraiment des infos et qu'on peut retarder notre départ

**Comment by Voisin1234** [comment]
Je valide ++
